app/controllers/seeds_controller.rb

```diff
@@ -109,4 +103,14 @@ def seed_params
       :days_until_maturity_min, :days_until_maturity_max, :organic, :gmo,
       :heirloom, :tradable_to, :slug)
   end
+
+  def seeds
```

Comment:
Might be clearer to pass these as arguments (owner = nil, crop = nil), or shift this onto the model as something "seeds_by_topic"